conjureup/controllers/configapps/gui.py

```diff
@@ -67,16 +67,11 @@ def generate_juju_machines(self):
         midx = 0
         for bundle_application in sorted(bundle.services,
                                          key=attrgetter('service_name')):
-            if bundle_application.placement_spec:
-                if app.provider.cloud == "localhost":
-                    app.log.info("Ignoring placement spec because we are "
-                                 "deploying to LXD: {}".format(
-                                     bundle_application.placement_spec))
-                else:
-                    app.log.warning("Ignoring placement spec because no "
-                                    "machines were set in the "
-                                    "bundle: {}".format(
-                                        bundle_application.placement_spec))
+            if not bundle_application.placement_spec:
```

Comment:
The `not` shouldn't be there, but I think we might as well just remove the entire warning.  I'm not even convinced it's true, anyway, since we don't actually strip out the placement directive and there's no real reason you couldn't say "put these two machines on 0" and have that just refer to the first automagically created machine.